### ISSUE TITLE
The hr in ghpages above the controls section is styled incorrectly

### ISSUE THREAD

**Issue by calebhearth** [Code52/metro.css#18]
The hr above the controls div in the ghpages demo appears too far left in Firefox 10.0.1 at 1920x1080.

Looked fine in Chrome at same resolution.

**Comment by shiftkey** [comment]
I'm looking to reproduce this on my side and can only see this style being applied:

```
hr {
    background-color: #1BA1E2;
    border: 0 none;
    height: 1px;
    width: 80%;
}
```
1. Can you confirm those styles are the ones being applied in Firebug?
2. Can you provide a screenshot of how it renders in FF to contrast?

Cheers,
@shiftkey

**Comment by ghost** [comment]
should be fixed by #20

**Comment by calebhearth** [comment]
@aeoth So it was. Looks good to me (I am on a different computer, but it's still Firefox 10.0.1).
